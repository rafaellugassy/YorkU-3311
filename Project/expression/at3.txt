forall
end_of_set_enumeration 
bool_value(true)
analyze
simplify 

restart

start_of_set_enumeration 
end_of_set_enumeration
analyze
simplify 