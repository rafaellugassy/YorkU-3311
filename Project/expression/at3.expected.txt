  Expression currently specified: ?
  Report: Expression is initialized.
->forall
  Expression currently specified: (&& ?)
  Report: OK.
->end_of_set_enumeration
  Expression currently specified: (&& ?)
  Report: Error (Set enumeration is not being specified).
->bool_value(True)
  Expression currently specified: (&& True)
  Report: OK.
->analyze
  Expression currently specified: (&& True)
  Report: (&& True) is not type-correct.
->simplify
  Expression currently specified: (&& True)
  Report: Error (Expression is not type-correct).
->restart
  Expression currently specified: ?
  Report: OK.
->start_of_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->end_of_set_enumeration
  Expression currently specified: {?}
  Report: Error: (Set enumeration must be non-empty).
->analyze
  Expression currently specified: {?}
  Report: Error (Expression is not yet fully specified).
->simplify
  Expression currently specified: {?}
  Report: Error (Expression is not yet fully specified).
